src/Resolution.ts

```diff
@@ -221,10 +221,21 @@ export default class Resolution {
    */
   async owner(domain: string): Promise<string | null> {
     domain = this.prepareDomain(domain);
-    const method = this.getNamingMethod(domain);
+    const method = this.getNamingMethodOrThrow(domain);
     return (await method.owner(domain)) || null;
   }
 
+  /**
+   * Custom key for the domain
+   * @param domain - domain name
+   * @param recordKey - key from resolver contract
```

Comment:
We should probably document it here that this method isn't implemented for ENS

Reply:
👌